README.md

```diff
@@ -96,6 +96,10 @@ To push without running the pre-push hook (e.g. doc updates):
 git push <remote> --no-verify
 ```
 
+### Release process
+(See [this doc](https://github.com/mozilla-mobile/firefox-tv/blob/master/.github/ISSUE_TEMPLATE/---relman-checklist.md) 
+for a description of our release process)
+
 ### Building release builds
 Release builds can be built in Android Studio or via the command line:
```

Comment:
nit:maybe we should comment official release builds are built on CI?

Reply:
Good thinking, I'll update.